salt/docker/init.sls

```diff
@@ -2,6 +2,61 @@ include:
   - repositories
   - flannel
 
+######################
+# additional ca.crt(s)
+#######################
+
+{% set registries = salt['pillar.get']('docker:registries', []) %}
```

Comment:
This would need to be updated to `{% set registries = salt['pillar.get']('registries', []) %}`